code/game/machinery/vendors/wardrobe_vendors.dm

```diff
@@ -573,3 +573,67 @@
 					/obj/item/storage/backpack/satchel_hyd = 50,
 					/obj/item/storage/backpack/duffel/hydro = 50)
 	refill_canister = /obj/item/vending_refill/hydrodrobe
+
+/obj/machinery/economy/vending/janidrobe
+	name = "\improper JaniDrobe"
+	desc = "A self cleaning vending machine capable of dispensing clothing for janitors."
+	icon_state = "janidrobe"
+	icon_lightmask = "base_drobe"
+	icon_panel = "drobe"
+	ads_list = list("Come and get your janitorial clothing, now endorsed by lizard janitors everywhere!")
```

Comment:
```suggestion
	ads_list = list("Come and get your janitorial clothing, now endorsed by janitors everywhere!")
```
it would be "unathi janitors" for us but i dont even think we have a culture of unathi players being jani mains.

Reply:
Good call, didn't check the description as I just copied it from the tg one